contracts/wrapper/NameWrapper.sol

```diff
@@ -431,19 +384,26 @@ contract NameWrapper is
      * @param _fuses initial fuses for the wrapped subdomain
      */
 
-    function setSubnodeOwnerAndWrap(
+    function setSubnodeOwner(
```

Comment:
Maybe we can delete this entirely? The only thing it does differently to `setSubnodeRecord` is set resolver and ttl; is it acceptable to just offer that interface alone?

Reply:
Hmm, it does take less gas though. I think it's acceptable to just offer that alone since it does everything the other function does. 

If you didn't want to set the resolver though, it would be quite a bit cheaper to use `setSubnodeOwner`.

Reply:
Fair enough. Let's leave it as-is.